### ISSUE TITLE
What does fintune mean?

### ISSUE THREAD

**Issue by laolongboy** [jingyuanli001/RFR-Inpainting#25]
Thanks for your contributions.
'For CelebA Dataset, train the model for 350,000 iterations and finetune for 150,000 iterations. (500,000 in total)'
I want to know what's the meaning of finetune in the same dataset.
According to the code, when finetuning,  parameters of BatchNorm2d will be fixed and the lr will change from 2e-4 to 5e-5.
Do I ignore some other operations ?

**Comment by jingyuanli001** [comment]
Your understanding regrading fine-tuning is completely correct. In fact, the main purpose of fine-tuning here is to stabilize the batch normalization layers which is not stable in inpainting tasks, due to the unpredictable mask shape and region. Without this procedure, the image color might be distorted.

**Comment by jingyuanli001** [comment]
You might refer to #33 to see what do the images look like before freezing the batch normalization.
